olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -122,6 +127,18 @@ struct CORE_API AuthenticationSettings {
    */
   TokenProviderCallback provider;
 
+  /**
+   * @brief The user-provided function that returns 'ApiKey'.
```

Comment:
```suggestion
   * @brief The user-provided function that returns `ApiKey`.
```
Please use the acute accent (`) not the apostrophe (').